Accordion.js

```diff
@@ -5,13 +5,14 @@ import React, {
 
 import {
   View,
+  ViewPropTypes,
   TouchableHighlight,
 } from 'react-native';
 
 import Collapsible from './Collapsible';
 
 const COLLAPSIBLE_PROPS = Object.keys(Collapsible.propTypes);
-const VIEW_PROPS = Object.keys(View.propTypes);
+const VIEW_PROPS = Object.keys(ViewPropTypes);
```

Comment:
To keep this backwards compatible you could do:
const VIEW_PROPS = (ViewPropTypes) ? Object.keys(ViewPropTypes) : Object.keys(View.propTypes);